Automattic/vip-go-mu-plugins
# e2e- Add video and trace recording to global setup for debugging assistance

## Description

This PR adds video recording and trace logging for the global setup script ( main tests already have this). It also causes the tests to fail fast if the login doesn't work so we're not wasting resources if we weren't able to log in.

I was also having an issue with running tests in cli and vs code and getting different results due to path. Made some changes to fix that.

## Checklist

Please make sure the items below have been covered before requesting a review:


## Steps to Test
Ensure tests pass in CI.

Run locally using `npm run setup-e2e-env` followed by `npm run test-e2e`
Make sure the tests pass and that there is a directory created with video and trace for the setup.
